qemu/tests/balloon_check.py

```diff
@@ -58,13 +58,14 @@ def get_ballooned_memory(self):
         return ballooned_mem
 
     @error_context.context_aware
-    def memory_check(self, step, ballooned_mem):
+    def memory_check(self, step, ballooned_mem, balloon_threshold=100):
         """
         Check memory status according expect values
 
         :param step: the check point string
         :type step: string
         :param ballooned_mem: ballooned memory in current step
+        :param balloon_threshold: guest memory compare fault-tolerant value                            
```

Comment:
trailing whitespace at the end of this line.
You may want to move this line under `:type ballooned_mem: int`
And add `:type` as well

Reply:
Modified,thanks